docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -289,13 +289,21 @@ Physical nodes (or cloud instances) are installed with a standard CentOS 7 serve
 1. Add a new `yugabyte:yugabyte` user and group with the default login shell `/bin/bash` that you set via the `-s` flag, as follows:
 
     ```bash
-    sudo useradd -s /bin/bash -m yugabyte   # (add user yugabyte and create /home/yugabyte)
+    sudo useradd -s /bin/bash --create-home --home-dir <yugabyte-home> yugabyte  # (add user yugabyte and create /home/yugabyte)
     sudo passwd yugabyte   # (add a password to the yugabyte user)
     sudo su - yugabyte   # (change to yugabyte user for execution of next steps)
     ```
 
+    `yugabyte-home` is the path to the Yugabyte home directory. You can omit this flag if you are using the default `/home/yugabyte`.
```

Comment:
nit: Please add a little more explanation to clarify that if you set a custom value for the yugabyte user's home in the yba UI, you need to use the same value here. If you left it empty, it can be omitted.